app/services/digest_sender.rb

```diff
@@ -0,0 +1,50 @@
+class DigestSender
+  DAILY_INTERVAL    = 1.day.ago
+  INSTANT_INTERVAL  = 10.minutes.ago
```

Comment:
I just noticed this change.
Setting "a time ago" in a constant is a tricky bug to catch later on. This means at the time the class is loaded `DAILY_INTERVAL` will get set to `1.day.ago`. And never change. It's a constant, and the `.ago` method is only applied when the constant is set.

This means if 1 day ago at load time is 17/11/2019. Then two weeks from today. DAILY_INTERVAL will still be 17/11/2019. Which is no longer a day ago, it's two weeks ago.

Anytime `ago` methods are used, they need to be dynamic to the time now (usually). This means we want to store the interval here which is 1.day. But never call `ago` until we want the differential.